### PR TITLE
feat: empty and loading states for tx history

### PR DESCRIPTION
**Pull request by Tulun**
Empty and loading states for tx history: 
Screen Shot 2022-04-19 at 10 54 02 AM
Screen Shot 2022-04-19 at 11 00 30 AM

### CODE REVIEW

File: src/views/Transactions/Transactions.tsx [hunk 1]
```diff
@@ -107,6 +114,22 @@ const Transactions = () => {
...
+              <LoadingWrapper>
+                <FontAwesomeIcon
+                  icon={faCircleNotch}
+                  className="fa-spin"
+                  size="2x"
+                />
+                <div>Loading...</div>
+              </LoadingWrapper>
+            )}
+            {!filledTx.length && !ongoingTx.length && !initialLoading && (
```
Comment (Gamaranto) [inline src/views/Transactions/Transactions.tsx:127]: nit: we can abstract this into its own variable like `isTxPresent`

**Review by Gamaranto — APPROVED** [review]
just a nit, otherwise lgtm

File: src/views/Transactions/TransactionsTable/MobileTransactionsTable.tsx [hunk 2]
```diff
@@ -30,84 +31,82 @@ const MobileTransactionsTable: FC<Props> = ({
...
-                        <div>{row.txHash}</div>
-                      </AccordionRow>
-                    </AccordionWrapper>
-                  )}
-                </>
-              );
-            })}
-          </StyledBody>
-        </StyledTableWrapper>
+      {rows.length && !initialLoading ? (
```
Comment (daywiss) [inline src/views/Transactions/TransactionsTable/MobileTransactionsTable.tsx:38]: imo the control of hiding/showing a component should happen in the parent unless its not being hidden fully

File: src/views/Transactions/TransactionsTable/TransactionsTable.tsx [hunk 3]
```diff
@@ -17,55 +17,58 @@ interface Props {
...
-                          </StyledCell>
-                        );
-                      })}
-                    </StyledRow>
-                  );
-                })
-              : null}
-          </StyledBody>
-        </StyledTableWrapper>
+      {rows.length && !initialLoading ? (
```
Comment (daywiss) [inline src/views/Transactions/TransactionsTable/TransactionsTable.tsx:31]: same here, it seems like the parent would be a better spot to hide the component

**Review by daywiss — APPROVED** [review]
looks good, id just address francescos comment

File: src/views/Transactions/TransactionsTable/TransactionsTable.tsx (hunk 3 above)
Comment (Tulun) [reply src/views/Transactions/TransactionsTable/TransactionsTable.tsx]: Fixed!

File: src/views/Transactions/Transactions.tsx (hunk 1 above)
Comment (Tulun) [reply src/views/Transactions/Transactions.tsx:127]: Fair, abstracted.

**Pull request merged**